franka_ros/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I now know this is because specific requirements exist for the cmake version used in a meta package (see https://wiki.ros.org/catkin/package.xml#Metapackages).